app/src/main/res/drawable/add_profile_edit_text_background_border.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <corners android:radius="4dp" />
+  <solid android:color="@color/white" />
+  <stroke
+    android:width="1dp"
+    android:color="@color/grey" />
+</shape>
```

Comment:
This file has been added but not getting used anywhere. Delete this.

Reply:
okay